src/components/callout/callout.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import Callout from './callout';
+import ShootingStar from '../../illustrations/shooting-star.svg';
+import ConstellationBigDipper from '../../illustrations/constellation-big-dipper.svg';
+
+const EXAMPLE_ILLUSTRATIONS = {
+  None: null,
+  ShootingStar: <ShootingStar />,
+  ConstellationBigDipper: <ConstellationBigDipper />,
+};
+
+export default {
+  component: Callout,
+  title: 'Components/Callout',
+  parameters: {
+    componentSubtitle:
+      'Display additional information detached from your surrounding text content',
```

Comment:
That's the sub title displayed in the docs. The actual description should be taken from the actual comment in the component but it's not picked up.

![Screen Shot 2021-11-15 at 17 02 21](https://user-images.githubusercontent.com/1701755/141814034-8939b68d-b2f3-4240-b9c7-e94c2fcfae0f.png)
: